helm-chart-sources/logstream-workergroup/README.md

```diff
@@ -33,6 +33,8 @@ This section covers the most likely values to override. To see the full scope of
 |config.host|"logstream-leader"|The resolvable hostname of your LogStream leader. |
 |config.rejectSelfSignedCerts|0| One of: `0` – allow self-signed certs; or `1` – deny self-signed certs. |
 |config.tlsLeader.enable|false|Enable TLS connectivity from the workergroup to its leader node |
+|config.hostNetwork|false|configures the workergroup to use the K8s host network instead of the container network.|
+|config.noProbe|false|turns off the liveness and readiness probs.|
```

Comment:
NIT wouldn't it be clearer without the negation, i.e. `config.probes|true` ?
also there's a typo in the description s/probs/probes/
I can fix both for you

Reply:
great feedback - thanks! I went ahead and fixed both issues in the branch.